server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/eventstore/CassandraEventStoreModule.java

```diff
@@ -38,8 +46,11 @@ protected void configure() {
 
         Multibinder.newSetBinder(binder(), CassandraModule.class)
             .addBinding()
-            .toInstance(org.apache.james.eventsourcing.eventstore.cassandra.CassandraEventStoreModule.MODULE);
+            .toInstance(org.apache.james.eventsourcing.eventstore.cassandra.CassandraEventStoreModule.MODULE());
 
-        Multibinder.newSetBinder(binder(), new TypeLiteral<EventDTOModule<?, ?>>() {});
+        Multibinder.newSetBinder(binder(), new TypeLiteral<EventDTOModule<? extends Event, ? extends EventDTO>>() {});
+        Multibinder.newSetBinder(binder(), new TypeLiteral<AdditionalInformationDTOModule<?, ? extends AdditionalInformationDTO>>() {});
```

Comment:
why did you need to add that?

Reply:
I will check it again, i add some issue with guice because i added some types.
Maybe i over did it here.